components/rhs_thread/rhs_thread.tsx

```diff
@@ -20,7 +23,7 @@ import RhsHeaderPost from 'components/rhs_header_post';
 import RhsRootPost from 'components/rhs_root_post';
 import FormattedMarkdownMessage from 'components/formatted_markdown_message';
 
-export function renderView(props) {
+export function renderView(props: Props) {
```

Comment:
Is this type correct? This function doesn't seem to take the same props as `RhsThread` since it's called by `react-custom-scrollbars`. Maybe that library provides its own type definition?